Types are broken with CJS/Typescript

```typescript
import MockAdapter from 'axios-mock-adapter';
```

This is transpiled into:
```js
const axios_1 = require("axios");
const axios_mock_adapter_1 = require("axios-mock-adapter");
// ...
const mockAxios = new axios_mock_adapter_1.default(axios_1.default);
```

Following the changes in fd982cd0fc, this no longer builds.

```
error TS1259: Module '".../node_modules/axios-mock-adapter/types/index"' can only be default-imported using the 'esModuleInterop' flag
index.d.ts(81, 1): This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -55,6 +55,8 @@ type RequestMatcherFunc = (
 ) => MockAdapter.RequestHandler;
 
 declare class MockAdapter {
+  static default: typeof MockAdapter;
+
   constructor(axiosInstance: AxiosInstance, options?: MockAdapterOptions);
 
   adapter(): AxiosAdapter;

--- a/types/test.ts
+++ b/types/test.ts
@@ -2,7 +2,8 @@ import axios from 'axios';
 import MockAdapter = require('axios-mock-adapter');
 
 const instance = axios.create();
-const mock = new MockAdapter(instance);
+let mock = new MockAdapter(instance);
+mock = new MockAdapter.default(instance)
 
 namespace AllowsConstructing {
   new MockAdapter(instance);

